apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -325,7 +327,7 @@ static void tc_libc_timer_strftime(void)
 	TC_ASSERT_EQ("strftime", atoi(buffer), st_time.tm_mon + 1);
 
 	/* Check the abbreviated month name */
-	strftime(buffer, BUFF_SIZE, "%b", &st_time);
+	strftime(buffer, BUFF_SIZE, "%h", &st_time);
```

Comment:
There is no special reason, it's changed to cover case 'h' also, as it FALL THROUGH to case 'b'